api/docs/code_reviews.dox

```diff
@@ -79,7 +79,16 @@ can select a particular reviewer if desired.
 
```

Comment:
> Some non-issue related commits prefix the summary with "iX:"

I was wondering where you had seen this: in fact there are 2 (and no more: exactly 2 out of 6K commits) that accidentally did this as new contributors.

I would re-word this line to say "There are 2 cases of commits that incorrectly prefix..." as otherwise this sounds like there are many or that this used to be the preferred policy or something.

I guess if you explicitly search for "iX:" and stop at the first match and never look at any other examples you might mistakenly conclude this is used extensively?  If you just look at the last 100 commits you see many examples with no prefix and zero examples of "iX:".

Reply:
Done.